### PR TITLE
new formula: globe

### PR DESCRIPTION
**Pull request by xdbr**
globe shows an ASCII graphic of the currently-lighted face of the Earth,
similar to phoon

### CODE REVIEW

File: Library/Formula/globe.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  desc "show an ASCII graphic of the currently-lighted face of the Earth"
+  homepage "http://www.acme.com/software/globe/"
+  url "http://www.acme.com/software/globe/globe_14Aug2014.tar.gz"
+  version "14Aug2014"
+  sha256 "5507a4caaf3e3318fd895ab1f8edfa5887c9f64547cad70cff3249350caa6c86"
+
+  def install
+    system "make"
+    bin.install "globe"
+    man1.install "globe.1"
```
Comment (mikemcquaid) [inline Library/Formula/globe.rb]: Is there no `make install` we can use?

File: Library/Formula/globe.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Globe < Formula
+  desc "show an ASCII graphic of the currently-lighted face of the Earth"
+  homepage "http://www.acme.com/software/globe/"
+  url "http://www.acme.com/software/globe/globe_14Aug2014.tar.gz"
+  version "14Aug2014"
```
Comment (mikemcquaid) [inline Library/Formula/globe.rb]: Maybe `0.0.20140814`?

**Comment by dunn** [comment]
@BrewTestBot test this please

File: Library/Formula/globe.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+class Globe < Formula
+  desc "show an ASCII graphic of the currently-lighted face of the Earth"
+  homepage "http://www.acme.com/software/globe/"
+  url "http://www.acme.com/software/globe/globe_14Aug2014.tar.gz"
+  version "0.0.20140814"
+  sha256 "5507a4caaf3e3318fd895ab1f8edfa5887c9f64547cad70cff3249350caa6c86"
+
+  def install
+    mkdir_p bin
+    mkdir_p man1
```
Comment (bfontaine) [inline Library/Formula/globe.rb]: Please use `bin.mkpath` and `man1.mkpath`.

**Comment by dunn** [comment]
From the bot:
```
==> audit problems
globe:
 * The homepage is not reachable (curl exit code 22)
```
Do you know if this is a temporary thing?

**Comment by bfontaine** [comment]
I think it’s temporary, I can access it from browser.

**Comment by xdbr** [comment]
@BrewTestBot test this please

**Comment by xdbr** [comment]
Any ideas why the build bot test on ee26cf1 fails?

Locally testing with 

```
brew install --verbose --debug globe && brew test --verbose --debug globe && brew audit --verbose --debug globe
```

runs just fine for me.

Also, the build on [this PR](https://github.com/Homebrew/homebrew/pull/41551) isn't too different to this one

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Comment by bfontaine** [comment]
The homepage is online but the server doesn’t implement `HEAD` and returns a `501` error when we attempt to do one, hence the failure.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/4c16da997de5040ff5bdf5191b93e3a225bbdada. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

Just a nit, but ideal commit message here would have been `globe 0.0.20140814 (new formula)`, Thanks!

**Pull request closed without merging**